aiohttp/web_urldispatcher.py

```diff
@@ -163,6 +163,11 @@ def __init__(self, name, prefix, directory, *,
             raise ValueError(
                 "No directory exists at '{}'".format(self._directory))
 
+        if sendfile_fallback or not hasattr(os, "sendfile"):
+            self.sendfile = self.sendfile_fallback
+        else:
+            self.sendfile = self.sendfile_system
```

Comment:
Better have `sendfile` method explicitly defined in class while it behaviour is controlled by just `hasattr(os, "sendfile")` - if system has sendfile support who will abuse it usage?